html-aam/index.html

```diff
@@ -191,6 +191,69 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
           the [[core-aam-1.2]].
         </p>
       </section>
+      <section id="mapping_contextual">
+        <h3>Exposing HTML Features With Contextual Roles</h3>
+        <p>
+          An element with a <strong>contextual role</strong> is one whose computed role semantics are based on its ancestral <a>accessibility tree</a> relationship to another element or elements. Or,
+          an element whose computed role is dependent upon whether it has other associated accessibility information, such as an author-provided accessible name.
+        </p>
+        <p>
+          For instance, an element that is an <strong>accessibility child</strong> of its required <strong>accessibility parent</strong> would expose a computed role for its given context, such as a
+          list item (`li` element) within an unordered list (`ul` element). However, the same element might be used within a different markup pattern and could therefore expose different
+          contextually-dependent computed roles, such as a `section` element with an author-provided accessible name (role `region`) versus an unnamed `section` element (role `generic`).
+        </p>
+        <p>When the conditions for an element's contextual role are not met, it will commonly expose a computed role of `generic` or with a <strong>minimum role</strong>, if applicable.</p>
+        <div class="note">
+          <p>Previously, the concept of a contextual role was loosely referred to as an element's "scoped" relationship to another element.</p>
+        </div>
+        <div class="example">
+          <p>
+            A <code>li</code> element has an implicit WAI-ARIA role mapping of <code>listitem</code> only when the element is an <strong>accessibility child</strong> of a <code>menu</code>,
+            <code>ol</code> or <code>ul</code> element that has not had its `list` role mapping suppressed.
+          </p>
+          <pre><code>&lt;ul>
+      &lt;li>...&lt;/li> &lt;!-- computed role `listitem` -->
+    &lt;/ul> 
+    &lt;article>
+      ...
+      &lt;li>...&lt;/li> &lt;!-- computed role `generic`, not `listitem` -->
+      ...</code></pre>
```

Comment:
Maybe we should use a different example than listitem, here and above, given the push back on the orphan roles tests which is partially being discussed here: https://github.com/w3c/aria/issues/2166

But started with the roll back of orphan `<li>` tests in WPT.

Reply:
orphan `li` elements was not where the pushback was.   it was orphan `element role=listitem` that had the pushback.

Reply:
Any time you're tempted to add an orphaned role test, please add a comment that points to the appropriate orphaned role-specific test file.

Reply:
> orphan li elements was not where the pushback was. it was orphan <element role=listitem> that had the pushback.

As I understand it, it's still the same issue whether the role is implicit or explicit.

Reply:
tried to message you this in slack, but i'm thinking of closing this PR down, and instead taking the approach of just clarifying the few elements that do have contextual roles mapping tables